zeek

```diff
@@ -34,47 +34,21 @@ else
 fi
 
 #The user can set the top level directory that holds all zeek content by setting it in "zeek_top_dir" (default "/opt/zeek")
-host_zeek=${zeek_top_dir:-/opt/zeek}
-
-host_zeek_logs="$host_zeek/logs"
-host_zeek_spool="$host_zeek/spool"
-host_zeek_etc="$host_zeek/etc"
-host_zeek_node_cfg="$host_zeek_etc/node.cfg"
-
-CONTAINER_NAME="zeek"
+HOST_ZEEK=${zeek_top_dir:-/opt/zeek}
 #Note, we force the 3.0 release for stability, though the user can override it by setting the "zeek_release" environment variable
-host_zeek_release=${zeek_release:-3.0}
-IMAGE_NAME="activecm/zeek:$host_zeek_release"
+IMAGE_NAME="activecm/zeek:${zeek_release:-3.0}"
+CONTAINER_NAME="zeek"
 
 # If the current user doesn't have docker permissions run with sudo
 SUDO=''
 if [ ! -w "/var/run/docker.sock" ]; then
 	SUDO="sudo --preserve-env "
 fi
 
-sudo --preserve-env mkdir -p "$host_zeek" "$host_zeek_logs" "$host_zeek_spool" "$host_zeek_etc"
-
-#See if we need to download the image first.  Note, the 3.0 release is the default via the IMAGE_NAME variable.
-if [ -z "`$SUDO docker images $IMAGE_NAME | grep -v '^REPOSITORY'`" ]; then
-	$SUDO docker pull --quiet "$IMAGE_NAME"
-fi
-
-if [ ! -s "$host_zeek_node_cfg" ]; then
-	echo "We do not appear to have a node.cfg file, so generating one now." >&2
-
-	sudo --preserve-env touch "$host_zeek_node_cfg"
-	$SUDO docker run --rm -it --network host \
-	 --mount source="$host_zeek_node_cfg",destination=/node.cfg,type=bind \
-	 "$IMAGE_NAME" \
-	 zeekcfg -o /node.cfg --type afpacket
-fi
```

Comment:
Moved this whole block to the "start" action. All others don't require it and it seems silly to first generate a node.cfg file when someone runs `zeek stop` or `zeek status`.